README.md

```diff
@@ -23,13 +23,34 @@ This project and its associated efforts are already monitoring tens of thousands
   [issues]: https://github.com/edgi-govdata-archiving/web-monitoring/issues
   [project_board]: https://github.com/orgs/edgi-govdata-archiving/projects/4
 
-- [Project Structure](#project-structure)
 - [Get Involved](#get-involved)
+- [Project Structure](#project-structure)
```

Comment:
I appreciate the desire to make the getting involved process more clear, BUT I disagree with this re-ordering.

Getting around and understanding the different repos involved in this project has been *demonstrably* hard for people, and making “project structure” the first item (several months ago things used to be in the order you are changing it back to here) has really improved things.

Reply:
oh ok! cool that we're iterating